### DISCUSSION TITLE
CSS files

### DISCUSSION THREAD

**Discussion by martinspeedie** [ben-rogerson/twin.examples#72] [Q&A]
Not really a bug, just a query, when I run Tailwind (with Styled Components) in my small NextJS project and run it on **dev** or **buid -> start** all the styles work perfectly.

However when I look in the **.next** folder I can't see any generated CSS files.

Where is the CSS?

**Comment by ben-rogerson** [comment]
The css lives in javascript files (css-in-js).
For more info about how this works, check out your chosen css-in-js library, eg: styled-components/emotion/stitches/goober
